### PR TITLE
Restore changes for better workflow

### PR DESCRIPTION
**Pull request by nikhil-achole**

### CODE REVIEW

File: tempest/api/workloadmgr/restore/test_image_booted.py [hunk 1]
```diff
@@ -1,323 +1,357 @@
...
+            raise Exception("Full snapshot failed")
+        return(snapshot_id)
+ 
+ 
+    @test.idempotent_id('9fe07175-912e-49a5-a629-5f52eeada4c9')
+    def test_1_image_booted(self):
+        try:
+            deleted = 0
+            ## VM and Workload ###
+            tests = [['tempest.api.workloadmgr.sanity.test_image_booted_Selective-restore',0], ['tempest.api.workloadmgr.sanity.test_image_booted_Inplace-restore',0], ['tempest.api.workloadmgr.sanity.test_image_booted_Oneclick-restore',0]]
```
Comment (savitha-p) [inline tempest/api/workloadmgr/restore/test_image_booted.py]: Replace 'sanity' with 'restore' in test names here

File: tempest/api/workloadmgr/restore/test_image_vol.py [hunk 2]
```diff
@@ -50,23 +69,45 @@ def calcmd5sum(self, flip, mount_point):
...
+
  
     @test.idempotent_id('9fe07175-912e-49a5-a629-5f52eeada4c9')
     def test_1_image_volume(self):
         try:
             global volumes
             deleted = 0
             ## VM and Workload ###
-            tests = [['tempest.api.workloadmgr.restore.test_image_vol_Selective-restore',0], ['tempest.api.workloadmgr.restore.test_image_vol_Inplace-restore',0], ['tempest.api.workloadmgr.restore.test_image_vol_Oneclick-restore',0]]
+            tests = [['tempest.api.workloadmgr.sanity.test_image_vol_Selective-restore',0], ['tempest.api.workloadmgr.sanity.test_image_vol_Inplace-restore',0], ['tempest.api.workloadmgr.sanity.test_image_vol_Oneclick-restore',0]]
```
Comment (savitha-p) [inline tempest/api/workloadmgr/restore/test_image_vol.py]: Replace 'sanity' with 'restore' here

Comment (nikhil-achole) [reply tempest/api/workloadmgr/restore/test_image_vol.py]: Done.

File: tempest/api/workloadmgr/restore/test_image_booted.py (hunk 1 above)
Comment (nikhil-achole) [reply tempest/api/workloadmgr/restore/test_image_booted.py]: Done

**Pull request merged**